docs/Developer Reference/link-token-contracts.md

```diff
@@ -12,7 +12,7 @@ metadata:
     0: '/files/72d4bd9-link.png'
 ---
 
-LINK tokens are used to pay node operators for retrieving data for smart contracts and also for deposits placed by node operators as required by contract creators.
+LINK tokens are used to pay node operators for retrieving data for smart contracts and also for deposits placed by node operators as required by contract creators. The smallest unit of LINK is called a Juel, and 1,000,000,000,000,000,000 (1e18) Juels are equal to 1 LINK.
```

Comment:
Is is it worth giving readers a reference mental model?
for eg:  "a Juels is to LINK what Wei is to Ether."

Reply:
Added